src/targets/registry.ts

```diff
@@ -93,6 +83,13 @@ export class RegistryTarget extends BaseTarget {
     githubRepo: GithubGlobalConfig
   ) {
     super(config, artifactProvider, githubRepo);
+    const remote = this.config.remote;
+    if (remote) {
+      const [owner, repo] = remote.split('/', 2);
+      this.remote = new GithubRemote(owner, repo);
+    } else {
+      this.remote = DEFAULT_REGISTRY_REMOTE;
+    }
     this.github = getGithubClient();
     this.githubRepo = githubRepo;
     this.registryConfig = this.getRegistryConfig();
```

Comment:
I see in many places that we're removing usage of `this.registryConfig` ... yet this remains, and I see that it is still used once, in `getValidItems`. Is that intentional?

Reply:
Yup. Since this happens in the constructor, we get an early warning/validation on the config by doing this.

One can argue `getValidItems` can be merged into `publish` as its only used there though.